src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,138 @@
+                    "     .'      .---.      '.\n" +
+                    "    /    .-----------.    \\\n" +
+                    "   /        .-----.        \\\n" +
+                    "   |       .-.   .-.       |\n" +
+                    "   |      /   \\ /   \\      |\n" +
+                    "    \\    | .-. | .-. |    /\n" +
+                    "     '-._| | | | | | |_.-'\n" +
+                    "         | '-' | '-' |\n" +
+                    "          \\___/ \\___/\n" +
+                    "       _.-'  /   \\  `-._\n" +
+                    "     .' _.--|     |--._ '.\n" +
+                    "     ' _...-|     |-..._ '\n" +
+                    "            |     |\n" +
+                    "            '.___.'\n" +
+                    "              | |\n" +
+                    "             _| |_\n" +
+                    "            /\\( )/\\\n" +
+                    "           /  ` '  \\";
+
+    /**
+     * Greeting and goodbye message constants
+     */
+    public static final String greetingMessage = "______________________________________\n" +
+            "Hello! I'm Handsome!\n" +
+            greetingArt +
+            "How can I help you?\n" +
+            "______________________________________\n";
+    public static final String goodbyeMessage = "Bye. Hope to see you again soon!\n" + goodbyeArt;
+
+    /**
+     * Commands
+     */
+    public static final String BYE = "bye";
+
+
+    /**
+     * Helper function to format message to be printed
+     */
+    public static final String DIVIDER = "______________________________________";
+
+    public static void printFormattedMessage(String message) {
+        System.out.println(DIVIDER);
+        System.out.println(message);
+        System.out.println(DIVIDER);
+
+    }
+
+    /**
+     * Prints greeting message
+     */
+
+    public static void printGreetingMessage() {
+        System.out.println(greetingMessage);
+    }
+
+    /**
+     * Prints exit message
+     */
+    public static void printGoodbyeMessage() {
+        System.out.println(goodbyeMessage);
+    }
+
+    /**
+     * New parser test
+     */
+
+    public static ExecutedCommand executeCommand(Command command) {
+        try {
+            return command.execute();
+        } catch (Exception e) {
+            return new ExecutedCommand(e.getMessage());
+        }
+    }
+
+    public static void main(String[] args) {
+
+        // Print greeting message
+        UI.printGreeting();
+
+        // Initialise instance of duke.tasks.TaskList
+        TaskList taskList = new TaskList();
+
+        System.out.println("Please enter your taskList command: (send 'bye' to exit)");
+        Scanner in = new Scanner(System.in);
+        /**
+         while (true) {
+         String userInput = in.nextLine();
+
+         TaskList.parseInput(userInput);
+
+         String[] parsedLine = userInput.split(" ", 2);
+
+         if (parsedLine[0].equals(BYE)) {
+         break;
+         }
+         } */
```

Comment:
Please avoid commenting out code blocks, it makes it a bit less readable. Consider removing this chunk of code if it isn't required.